### PR TITLE
[Hotfix] JWE

### PR DESCRIPTION
**Pull request by chrisseto**
Use JWE with JWTs

corresponds with  https://github.com/CenterForOpenScience/waterbutler/tree/hotfix/0.13.2

Note: Additional settings are required for this PR

### CODE REVIEW

File: website/settings/defaults.py [hunk 1]
```diff
@@ -306,6 +306,9 @@ def parent_dir(path):
         },
     }
 
+WATERBUTLER_JWE_SALT = 'yusaltydough'
```
Comment (sloria) [inline website/settings/defaults.py]: Should these settings be added to set of required settings in `settings/__init__.py`?

**Comment by sloria** [comment]
Does this solve a specific bug?

File: website/settings/defaults.py (hunk 1 above)
Comment (chrisseto) [inline website/settings/defaults.py]: They are

Comment (sloria) [inline website/settings/defaults.py]: Ah, sure enough.

**Pull request merged**